### PR TITLE
ORCT-126 Flags assignment dialog

### PR DESCRIPTION
**Pull request by Ehevi**
### I have a JIRA ticket
- [x] branch and/or PR name(s) include(s) JIRA ID
- [x] issue has "Fix version" assigned
- [x] issue "Status" is set to "In review"
- [x] PR labels are selected

### Notable changes for users:
- QC type selection modal

### Notable changes for developers:
- navigation regactor
- link chip refactor (`actionChip` -> `linkChip` + component cleanup :broom:)

### CODE REVIEW

File: app/config/rct-data/quality/operations.js
Comment (xsalonx) [inline app/config/rct-data/quality/operations.js:1]: unnecessary

Comment (Ehevi) [reply app/config/rct-data/quality/operations.js:1]: removed

**Review by xsalonx — COMMENTED** [review]
Access to time based qualities from RunsPerPeriod view should be disabled

File: app/public/model/PrimaryModel.js [hunk 1]
```diff
@@ -183,15 +119,10 @@ export default class PrimaryModel extends Observable {
     removeCurrentData() {
         const { page, index } = this.getCurrentDataPointer();
         this.fetchedData[page][index] = null;
-        this.goToDefaultPageUrl(page);
```
Comment (xsalonx) [inline app/public/model/PrimaryModel.js:186]: Why this way?

File: app/public/views/runs/runsPerDataPass/overview/content.js [hunk 2]
```diff
@@ -0,0 +1,139 @@
...
+                                if (Array.from(modalContent).find((e) => e != event.target)
+                                && Array.from(modal).find((e) => e == event.target)
+                                && document.getElementById('pageSettingsModal')) {
+                                    document.getElementById('pageSettingsModal').style.display = 'none';
+                                }
+                            });
+                        },
+                    }, h('.settings-20'))),
+
+                h('div', functionalities(model))),
```
Comment (xsalonx) [inline app/public/views/runs/runsPerDataPass/overview/content.js:101]: I think that replacing operands will increase readability

**Review by xsalonx — APPROVED** [review]

**Pull request merged**